test/gtest/ucs/test_async.cc

```diff
@@ -222,12 +239,20 @@ class local_event : public local,
 {
 public:
     local_event(ucs_async_mode_t mode) : local(mode), base_event(mode) {
+        set_event();
+    }
+
+    void set_event() {
         set_handler(&m_async);
     }
 
-    ~local_event() {
+    void unset_event() {
```

Comment:
why set_event/unset_event wrappers?

Reply:
1. `m_async` is proteted member of `base_event`, cannot be accessed inside test case.
2. If rename `set_event()` to `set_handler()`, it will ***overshadow*** the `set_handler()` of `base_async`, which might make method confusing.